core-services/prow/02_config/_plugins.yaml

```diff
@@ -245,6 +245,12 @@ bugzilla:
       - 4.5.z
       target_release: 4.4.z
       validate_by_default: true
+    openshift-4.5:
+      dependent_bug_target_releases:
+      - 4.6.0
+      - 4.6.z
+      target_release: 4.5.z
```

Comment:
I *think* until the relesae this should be `4.5.0`? But I'm not sure - I've seen @sdodson knowing stuff like this :)

Reply:
Yeah this should be 4.5.0 until we cut final GA build.